cmd/swapd/main.go

```diff
@@ -352,14 +358,24 @@ func (d *daemon) make(c *cli.Context) error {
 		return err
 	}
 
+	dbCfg := &chaindb.Config{
+		DataDir: path.Join(cfg.DataDir, "db"),
```

Comment:
Before we merge this PR, can you add this to the `default-file-locations.md` document? Include information about the database, like whether it is for internal use by our software only, or if the user might pull information from it, and when the end-use can safely delete it.

Reply:
yes, for sure!